security.md

```diff
@@ -0,0 +1,20 @@
+# Security policy
```

Comment:
As mentioned before I think we can improve the policy but for sure starting by having this documents is a clear progress and better that do not have no policy at all.